projects/addon-doc/components/theme-switcher/theme-switcher.template.html

```diff
@@ -1,11 +1,22 @@
-<tui-select
-    [ngModel]="theme"
-    (ngModelChange)="onTheme($event)"
+<tui-textfield
+    tuiChevron
+    [tuiTextfieldCleaner]="false"
 >
-    <ng-content />
-    <tui-data-list *tuiDataList>
+    <label tuiLabel>
+        <ng-content />
+    </label>
+    <input
+        tuiSelect
+        [ngModel]="theme"
+        (ngModelChange)="onTheme($event)"
+    />
+    <tui-data-list
+        *tuiTextfieldDropdown
+        new
```

Comment:
Do we actually need `new` for `<tui-data-list />` ? Or only for `[tuiOption]` ?

Reply:
This is main so we will remove all "new" attrs soon anyway